server/nix/default.nix

```diff
@@ -27,7 +27,7 @@ pkgs.haskell-nix.cabalProject {
       cardano-prelude
       optparse-applicative
       plutus-tx
-      plutus-ledger-api
```

Comment:
This package should be added back to `additional`

Reply:
This was removed accidentally instead of plutus-tx-plugin.